public/assets/models/winter_high_salt.json

```diff
@@ -0,0 +1,10 @@
+{
+  "slices":55,
+  "extension":".png",
```

Comment:
it looks the variables not encoded in the file name already are all the same across all of these `.json` files.  Are they really needed? Will there be variation in the future? Or can these assumptions be moved closer to the code?

Reply:
On this case, the bay data size and number or slices will not change. It could not be the case for other data needs to be integrated to the viewer. I am not a friend of hardcoding specific data in the code. Maybe we can have just a single json with all the info that needs to be loaded?
```

{
  "slices" : 55,
   seasons:["winter","summer"] ,
   tides:["high","low"],
   variables:["temp","salt"]
}
```

Reply:
I like that approach.  This could also be used in the toggle code @RobertGemmaJr so that those values don't need to be hardcoded either and the first of each array was the default.

Reply:
So the toggle buttons would be `obj.seasons[0]` and `obj.seasons[1]`? Once this goes through I could tackle that

Reply:
Almost.  So the initial value would be set to `seasons[0]` and then the buttons would be added in a for loop.  This way if a value is added to the `config.json` then the toggles will automatically update.

Reply:
Oh yes that makes sense, smart!